### ISSUE TITLE
Version schema

### ISSUE THREAD

**Issue by remicollet** [nbs-system/snuffleupagus#135]
For discussion:

The "make release" defines `PHP_SNUFFLEUPAGUS_COMMIT` which can seems a bit strange (some could expect this to be defined for git build, not for "release" (tagged version)

More the (always present) space in version raise strange comparison issue

```php
$  php -r 'var_dump(phpversion("snuffleupagus"), version_compare(phpversion("snuffleupagus"), "0.2", ">="));'
string(4) "0.2 "
bool(false)
```

true is obviously expected

Proposal, in php_snuffleupagus.h (using 3 digits, per common practice, and matching tag schema)
`#define PHP_SNUFFLEUPAGUS_VERSION "0.2.0" `

In MINFO
`php_info_print_table_row(2, "Version", PHP_SNUFFLEUPAGUS_VERSION " " PHP_SNUFFLEUPAGUS_COMMIT);`

**Comment by jvoisin** [comment]
I don't think that it's worth having the commit hash in the release number anyway, since either people will use releases, or are clever enough to give use the hash if needed.

**Comment by remicollet** [comment]
So, possible fix on #136

**Comment by jvoisin** [comment]
The [documentation on PECL](https://pecl.php.net/account-request.php) states that "Note: wrappers for GPL (all versions) or LGPLv3 libraries will not be accepted. Wrappers for libraries licensed under LGPLv2 are however allowed while being discouraged.". We're not a wrapper, so I don't know if we can't be included in PECL.

**Comment by jvoisin** [comment]
The [documentation on PECL](https://pecl.php.net/account-request.php) states that "Note: wrappers for GPL (all versions) or LGPLv3 libraries will not be accepted. Wrappers for libraries licensed under LGPLv2 are however allowed while being discouraged.". We're not a wrapper, so I don't know if we can't be included in PECL.

Some [packages]( https://pecl.php.net/package/spplus ) are LGPL.

**Comment by blotus** [comment]
Closed by cb613937dceda3e9768b96a67e2a6b6cad800c24
